modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ddl/DdlToCatalogCommandConverter.java

```diff
@@ -88,6 +90,9 @@ static CatalogCommand convert(CreateZoneCommand cmd) {
         String engine = Objects.requireNonNullElse(cmd.dataStorage(), DEFAULT_STORAGE_ENGINE);
         String dataRegion = (String) cmd.dataStorageOptions().getOrDefault("dataRegion", DEFAULT_DATA_REGION);
 
+        // TODO: https://issues.apache.org/jira/browse/IGNITE-20990 get rid of dummy storages.
+        String storageProfiles = cmd.storageProfiles() == null ? DUMMY_STORAGE_PROFILE : cmd.storageProfiles();
```

Comment:
Why do we need default here? In the DdlSqlToCommandConverter, for example, we throw an error, if the storageProfiles string is empty